src/administrative-sdk/speech-challenge/speech-challenge-controller.js

```diff
@@ -18,20 +18,26 @@ export default class SpeechChallengeController {
 
   /**
    * Create a speech challenge in the current active {@link Organisation} derived from the OAuth2 scope.
+   * Additional information like audio, .srt files or images can be added in HTML5 Blob format.
+   * The returned SpeechChallenge will contain links to download the given files.
    *
    * @param {SpeechChallenge} speechChallenge - Object to create.
    * @param {?Blob} audioBlob - Audio fragment to link to the challenge.
+   * @param {?Blob} srtFile - SRT file in HTML5 Blob format to accompany the challenge.
+   * @param {?Blob} image - Image file in HTML5 Blob format to accompany the challenge.
    * @returns {Promise.<PronunciationChallenge>} Promise containing the newly created SpeechChallenge.
    * @throws {Promise.<Error>} If the server returned an error.
    */
-  createSpeechChallenge(speechChallenge, audioBlob) {
+  createSpeechChallenge(speechChallenge, audioBlob, srtFile, image) {
```

Comment:
I think we could or should be using object destructoring more. It saves long argument lists and makes it more easy to write pure functions.